mesonbuild/interpreter.py

```diff
@@ -3733,6 +3733,13 @@ def func_add_test_setup(self, node, args, kwargs):
         timeout_multiplier = kwargs.get('timeout_multiplier', 1)
         if not isinstance(timeout_multiplier, int):
             raise InterpreterException('Timeout multiplier must be a number.')
+        is_default = kwargs.get('is_default', False)
+        if not isinstance(is_default, bool):
+            raise InterpreterException('Is_default option must be a boolean')
+        if is_default:
+            if self.build.test_setup_default_name is not None:
+                raise InterpreterException('Is_default can be set to true only once')
```

Comment:
This should print what the pre-existing default test setup is for easier debugging

Reply:
:+1: